### ISSUE TITLE
Typing non-ascii characters doesn't work

### ISSUE THREAD

**Issue by Tuomoz** [01org/fMBT#52]
It seems that typing non-ascii characters like 'ö' and 'ä' is not working at least with the X11 and Windows platforms. For example:
```Python
import fmbtx11
screen = fmbtx11.Screen()
screen.type(u'aäa')
```
Simulates only the two a-characters and skips the ä-character. Are there plans to add support for typing arbitrary unicode characters?

**Comment by askervin** [comment]
Checked out how to do it on X11. The first method that I tried takes a bit manual work and reading X11/keysymdef.h, but seems to work. ``fmbtx11.Screen().type(u"åäöÅÄÖ")`` works on commit e7d1092a4c2f8b9ce5a87b079875300ad2c9d7e5.

To answer your question: support is not on TODO list at the moment, but definitely I'll take it in for Windows as well, if it would be as easy as this.

On platforms where typing is implemented by simulating a USB keyboard (fmbtuinput.Keyboard()) this would be much harder, because it depends on keyboard layouts used by upper software layers.

**Comment by askervin** [comment]
Checked out Windows as well. ``fmbtwindows.Device(hostspec).type(u"åäöÅÄÖ")`` works on commit 94fc92291a2c7c7fe465e4e079c1c1395af6e4d5, given that you can easily (without dead key) type the letters using your current keyboard layout.

Closing the issue.

**Comment by mixu-** [comment]
If you need to input characters not on your keyboard layout, you can probably use the clipboard functionality available since 0.33:

_"Device.getClipboard() and setClipboard(data) added to enable clipboard usage on test scripts. Currently text data format is supported."_

**Comment by Tuomoz** [comment]
Thank you for your very fast fix! I just tested typing the 'ä' and 'ö' characters on both X11 and Windows and it's working correctly. Also thanks for the tip regarding clipboard.
